boards/opentitan/README.md

```diff
@@ -109,23 +109,80 @@ export OPENTITAN_TREE=/home/opentitan/
 
 then you can run `make flash` or `make test-hardware` to use the board.
 
-### Compiling the Kernel for FPGA or Verilator
+Verilator
+---------
 
 Opentitan is supported on both an FPGA and in Verilator. Slightly different
 versions of the EarlGrey chip implementation are required for the different
-platforms. By default the kernel is compiled for the FPGA. To compile for
-Verilator, run:
+platforms. By default the kernel is compiled for the FPGA.
+
+## Setting up Verilator
+
+For a full guide see the official OpenTitan documentation: https://docs.opentitan.org/doc/ug/getting_started_verilator/
+
+A quick summary on how to do this is included below though
+
+### Build FuseSoc
+
+```shell
+git clone https://github.com/lowRISC/opentitan.git
+cd opentitan
+git checkout <OpenTitan_SHA>
+pip3 install --user -r python-requirements.txt
+
+LANG="en_US.UTF-8" fusesoc --cores-root . run --flag=fileset_top --target=sim --setup --build lowrisc:dv:chip_verilator_sim
+```
+
+### Test Verilator
+
+```shell
+build/lowrisc_dv_chip_verilator_sim_0.1/sim-verilator/Vchip_sim_tb \
+    --meminit=rom,./build-out/sw/device/boot_rom/boot_rom_sim_verilator.scr.39.vmem \
+    --meminit=otp,./build-bin/sw/device/otp_img/otp_img_sim_verilator.vmem
+
+# Read the output, you want to attach screen to UART
+screen /dev/pts/4
+
+# Wait a few minutes
+# You should eventually see messages in screen
+# Once you see "waiting for SPI input..." you know it works
+```
+
+### Build and Run Tock
+
+To compile Tock for Verilator, run:
 
 ```shell
 make BOARD_CONFIGURATION=sim_verilator
 ```
 
-To explicitly specify the FPGA, run:
+You will then need to generate a vmem file:
+
+```shell
+srec_cat \
+    target/riscv32imc-unknown-none-elf/release/earlgrey-cw310.bin \
+    --binary --offset 0 --byte-swap 8 --fill 0xff \
+    -within target/riscv32imc-unknown-none-elf/release/earlgrey-cw310.bin\
+    -binary -range-pad 8 --output binary.64.vmem --vmem 64
+```
+
+And Verilator can be run with:
+
+```shell
+${OPENTITAN_TREE}/build/lowrisc_dv_chip_verilator_sim_0.1/sim-verilator/Vchip_sim_tb \
+    --meminit=rom,${OPENTITAN_TREE}/build-out/sw/device/boot_rom/boot_rom_sim_verilator.scr.39.vmem \
+    --meminit=flash,./binary.64.vmem \
+    --meminit=otp,${OPENTITAN_TREE}/build-bin/sw/device/otp_img/otp_img_sim_verilator.vmem
+````
+
+You can also use the Tock Make target to automatically build Tock and run it with Verilator
 
 ```shell
-make BOARD_CONFIGURATION=fpga_nexysvideo
+make BOARD_CONFIGURATION=sim_verilator verilator
```

Comment:
This sort of suggests that this will work for either OT target, but the Makefile changes to support verilator are only implemented for the cw310